src/firefox/ffBrowser.ts

```diff
@@ -168,6 +168,11 @@ export class FFBrowserContext extends BrowserContextBase {
   }
 
   async _initialize() {
+    if (this._options.locale) {
+      if (!this._options.extraHTTPHeaders)
+        this._options.extraHTTPHeaders = {};
+      this._options.extraHTTPHeaders['Accept-Language'] = this._options.locale;
```

Comment:
I think we should leave `options.extraHTTPHeaders` as is and instead update FFBrowserContext.setExtraHTTPHeaders.

Reply:
Done